core/state_processor.go

```diff
@@ -138,9 +129,7 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 		requests = [][]byte{}
 	}
 
-	if isJovian && *usedGas < daFootprint {
-		*usedGas = daFootprint
-	}
+	*usedGas = CalculateGasUsed(block.Header(), block.Transactions(), *usedGas, p.config, p.chain.GetHeaderByNumber(header.Number.Uint64()-1))
```

Comment:
kPossible underflow for Jovian at genesis. Also need to take a closer look around activation block.